proto/src/determined/run/v1/run.proto

```diff
@@ -120,3 +120,118 @@ message FlatRun {
   // Project level local id of run.
   string local_id = 22;
 }
+
+// Flat run respresentation.
+message Run {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: {
+      required: [
+        "id",
+        "start_time",
+        "state",
+        "tags",
+        "checkpoint_size",
+        "checkpoint_count",
+        "project_id",
+        "project_name",
+        "workspace_id",
+        "workspace_name",
+        "parent_archived",
+        "archived"
+      ]
+    }
+  };
+  // The id of the run.
+  int32 id = 1;
+  // The time the run was started.
+  google.protobuf.Timestamp start_time = 2;
+  // The time the run ended.
+  google.protobuf.Timestamp end_time = 3;
+  // The current state of the run(trial).
+  determined.trial.v1.State state = 4;
+  // The tags of the associated experiment.
+  // TODO(aaron.amanuel): Create add/remove tags for runs.
```

Comment:
Is this worth tracking in a Jira ticket, or appending as a comment or description to something we're doing soon?

Reply:
tags are still on the experiment/search level. So it would be a larger effort to move it down. I'll remove this TODO though as it was copied over from flat runs object